api/proto/info/info.proto

```diff
@@ -918,3 +920,13 @@ message ZInfoAppInstMetaData {
   AppInstMetaDataType type = 2;
   bytes data = 3;
 }
+
+// ZInfoEdgeview - edge-view operation status/event
+message ZInfoEdgeview {
+  // expires on timestamp
+  google.protobuf.Timestamp expire_time = 1;
+  // started timestamp
+  google.protobuf.Timestamp started_time = 2;
+  // total number of cmds edge-view has processed
+  uint32 count = 3;
```

Comment:
Would it make sense to have a separate counter for app commands and dev commands to get better visibility?

Reply:
yes, will change to that